src/main/kotlin/graphics/scenery/Settings.kt

```diff
@@ -20,7 +20,7 @@ open class Settings(override var hub: Hub? = null) : Hubable {
      * @param[name] The name of the setting
      * @return The setting as type T
      */
-    fun <T> get(name: String): T {
+    operator fun <T> get(name: String): T {
```

Comment:
I'd like to keep them as explicit get/set, mainly due to the template parameters necessary sometimes.

Reply:
The thing is, you can still use them as explicit.

Although, I think that 

`val value: Int = settings["ciao"]`

is pretty self-explanatory 

Moreover, when the template parameter is necessary, you cannot use the operator

but your project, your rules